### DISCUSSION TITLE
Activate slider for mobile only

### DISCUSSION THREAD

**Discussion by ottovw** [naver/egjs-flicking#757] [Q&A] [answered]
Hi,

We're using flicking with react, nextjs. Thanks for the great lib.

we have 3 cols which we want to show when visiting on desktop or larger screens:
![image](https://user-images.githubusercontent.com/1045946/204093866-cbb1e19c-194f-4acd-9141-780abe0788c6.png)

and we only want the slider to show up on mobile or small screen devices:
![image](https://user-images.githubusercontent.com/1045946/204093917-70af51b6-a2f4-4b88-bdac-dd8fdfa2cec7.png)

is there a responsive way of doing this? or do you need a js based mobile detection and swap the wrap the react components?

**Comment by malangfox** [comment, marked as answer]
Hello @ottovw. I'm glad and happy you like our component.

Flicking does not support mobile detection separately, but you can enable/disable whether Flicking moves with user input depending on the width of the window.
I think you want Flicking to move by user input and `panelsPerView` to be 1 only on small screen widths.
This can be implemented using the `beforeResize` event. [You can check out this demo.](https://codepen.io/malangfox/pen/poKKOQr)
This demo set the layout and behavior of Flicking by initial width of the screen and update them when screen size changes.

If you have additional questions, please feel free to leave a comment.

**Reply by ottovw** [reply]
Thanks!
